lua/plenary/benchmark/init.lua

```diff
@@ -0,0 +1,104 @@
+local B = {}
+local stat = require "plenary.benchmark.stat"
+
+local get_stats = function(results)
+  local ret = {}
+
+  ret.max, ret.min = stat.maxmin(results)
+  ret.mean = stat.mean(results)
+  ret.median = stat.median(results)
+  ret.std = stat.std_dev(results)
+
+  return ret
+end
+
+local get_output = function(index, res, runs)
+  -- divine with a sutable one / 1e3, 1e6, 1e9
```

Comment:
This part is still an issue. Right now `/ 1e6` is fix. for the io run this isnt good. as you might have seen

Reply:
so here we might wanna do `/ 1e3` and we wanna determine this dynamic. Maybe we have a function that takes seconds to complete so we want `1e9`

Reply:
helped you out :laughing: 

![image](https://user-images.githubusercontent.com/15233006/133000438-ff5e96f5-2502-440a-8181-dea81a5052a1.png)

[patch.diff](https://gist.github.com/Conni2461/599a975e4d7a64a7899eccd4347a0223)

Reply:
proving that my diff actually works

![image](https://user-images.githubusercontent.com/15233006/133000556-0d092b52-b6cd-4df8-9a2b-2f10a9a70c55.png)